apps/example-app/src/background.js

```diff
@@ -151,9 +151,8 @@ addEventListener("remoteNotification", (resolve, reject, args) => {
   resolve();
 });
 
-addEventListener("checkWatchReachability", (resolve, reject, args) => {
```

Comment:
The event listener function signature needs to be: `(resolve, reject, args) => void`